frequency: 5.0
scan_topic: /burger/scan
human_topic: /pedsim_agents/semantic/pedestrian
human_frame: map
robot_frame: base_link

add_noise: true
mean: 0.0
std_dev: 1.0

num_particles: 100
